aviatrix/resource_aviatrix_cloudn_transit_gateway_attachment.go

```diff
@@ -172,16 +215,27 @@ func resourceAviatrixCloudnTransitGatewayAttachmentRead(ctx context.Context, d *
 	d.Set("enable_over_private_network", attachment.EnableOverPrivateNetwork)
 	d.Set("enable_jumbo_frame", attachment.EnableJumboFrame)
 	d.Set("enable_dead_peer_detection", attachment.EnableDeadPeerDetection)
+	d.Set("enable_learned_cidrs_approval", attachment.EnableLearnedCidrsApproval)
+	d.Set("approved_cidrs", attachment.ApprovedCidrs)
```

Comment:
approved_cidrs is a type of set, which is not a primitive type. Add a check for setting it.